resources/js/Pages/Welcome.vue

```diff
@@ -0,0 +1,386 @@
+                                fill="currentColor"
+                            />
+                        </svg>
+                    </div>
+                    <nav v-if="canLogin" class="-mx-3 flex flex-1 justify-end">
```

Comment:
Consider using a centralized store or context for managing user authentication state instead of passing props like `canLogin` and `canRegister` deep into components. This approach can simplify prop management and improve component reusability. [important]